worker/training.ts

```
@@ -0,0 +1,548 @@
+    // モデル学習（簡易版）
+    let modelData: ArrayBuffer
+    let metrics: TrainingResult['metrics']
+    
+    switch (config.task) {
+      case 'master_reg':
+        const regResult = await trainRegressionModel(trainX, trainY, valX, valY, config)
+        modelData = regResult.model
+        metrics = regResult.metrics
+        break
+        
+      case 'preset_cls':
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)